### PR TITLE
Fix storing of dockerfile artifact

### PR DESCRIPTION
**Pull request by harshad16**
Fix storing of dockerfile artifact
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Fixes: #100 

## This introduces a breaking change
- [x] No

## This Pull Request implements

The copying content of Dockerfile doesn't require bash error checking.

### CODE REVIEW

File: amun/base/argo-workflows/inspection-build-with-cpu.yaml [hunk 1]
```diff
@@ -234,13 +234,16 @@ spec:
...
           mkdir -p "$results"
           kubectl logs "{{inputs.parameters.inspection-id}}-1-build" > "${results}/log"
-          cat <<EOFEOF  | jq . > "${results}/specification"
+          cat << EOF  | jq . > "${results}/specification"
           {{inputs.parameters.specification}}
-          EOFEOF
-          cat <<EOFEOF > "${results}/Dockerfile"
-          {{inputs.parameters.dockerfile}}
-          EOFEOF
+          EOF
```
Comment (goern) [inline amun/base/argo-workflows/inspection-build-with-cpu.yaml:239]: is this EOF intended?

Comment (fridex) [reply amun/base/argo-workflows/inspection-build-with-cpu.yaml:239]: I think so, it's corresponding to the `cat << EOF`.

File: amun/base/argo-workflows/inspection-build-with-cpu.yaml [hunk 2]
```diff
@@ -234,13 +234,16 @@ spec:
...
 
+          set +e
```
Comment (fridex) [inline amun/base/argo-workflows/inspection-build-with-cpu.yaml:241]: What was the issue here?

Comment (fridex) [reply amun/base/argo-workflows/inspection-build-with-cpu.yaml:241]: Ah, I see. It's a fix to #100. Very nice finding! :100:

Comment (fridex) [reply amun/base/argo-workflows/inspection-build-with-cpu.yaml:241]: I wonder why bash does not treat all the input as a string until EOF though.

**Pull request merged**